### ISSUE TITLE
[PSA] Centralize the info on shortcuts

### ISSUE THREAD

**Issue by have-fun-was-taken** [have-fun-was-taken/yafc-ce#171]
It's awesome that YAFC has shortcuts. Now we just need to tell the users which shortcuts exist. I think of an addition to the Docs files of the repository would be a good start.

**Comment by have-fun-was-taken** [comment]
Yes, I think a good idea would be to expand the startup tips with the new shorcuts from #154

**Comment by DaleStan** [comment]
I don't know how much it counts, but when you hover over a button with a right-click shortcut, a tooltip appears. Here are two of them, though you'll only see one at a time in real life.
![image](https://github.com/have-fun-was-taken/yafc-ce/assets/21223975/eaf926a5-e0dd-4ec4-8e24-b3d8159b62ef)

There are no hints for Enter or Escape, though.

**Comment by shpaass** [comment]
Criterion of ticket completion: 
Either me or someone else needs to go over all the shortcuts and check that they are explained at least somewhere somehow.

**Comment by DaleStan** [comment]
Sunrosa reported that MainScreen.cs contains "ctrl+s = save; ctrl+z = undo; ctrl+y = redo; ctrl+n = neie; ctrl+f = page search; ctrl+shift+f = all search; ctrl+t = new page; ctrl+(shift)+tab = switch page; ctrl+pgup/pgdown = switch page; ctrl+shift+pgup/pgdown = move page" (This sounds complete to me, so I didn't double-check.)

Elsewhere, I know of or found:
* All the keys you'd expect (except redo, ctrl-(shift-)left/right, and double-click) in edit boxes, plus ctrl-backspace. (OK, maybe you expected this one, but I didn't. However, it even works in Notepad, so it probably belongs on the expected list.)
* Most dialogs can be dismissed with escape or enter/return.
* Backspace often works as "Back" (but not in NEIE, for reasons that are at least justifiable if not good)
* Many (all?) scrollable areas can be scrolled vertically with pgup/pgdn/home/end and scrolled horz&vert with the arrow keys.

All of Sunrosa's are listed in at least one of Tips.txt or the UI. With the current UI, I'd argue that none of the rest need to be documented. That said, if back buttons are added to Dependency Explorer and Link Summary, they should have tooltips; and if edit boxes get a right-click menu, cut/copy/paste and select all should have their shortcuts listed.

**Comment by veger** [comment]
> Many (all?) scrollable areas can be scrolled vertically with pgup/pgdn/home/end and scrolled horz&vert with the arrow keys.

Scroll horizontal with ctrl+scroll wheel
